### ISSUE TITLE
Bug - having troubles building with parcel bundler

### ISSUE THREAD

**Issue by seanforyou23** [patternfly/patternfly-react#9624]
**Describe the problem**
Trying to use PF5 with parcel. Getting the following after importing `@patternfly/react-core`

```
🚨 Build failed.

@parcel/transformer-css: Unexpected token Ident("false")

  /path/to/project/node_modules/@patternfly/react-styles/css/components/Banner/banner.css:118:14
    117 | :where(.pf-v5-theme-dark) .pf-v5-c-banner {
  > 118 |   color: var(false);
  >     |              ^
    119 | }
    120 | :where(.pf-v5-theme-dark) .pf-v5-c-banner.pf-m-blue, :where(.pf-v5-theme-dark) .pf-v5-c-banner.pf-m-red, :where(.pf-v5-theme-dark) .pf-v5-c-banner.pf-m-green, :where(.pf-v5-theme-dark) .pf-v5-c-banner.pf-m-gold {
```

**How do you reproduce the problem?**
I don't have a minimal reproduction of the issue at the moment.

**Expected behavior**
To be able to import patternfly and use the components in a basic sense.

**Is this issue blocking you?**
Not blocking anything, but preventing me from integrating PF in an existing project (it's currently using material-ui)

**What is your environment?**
 - OS: MacOS
 - Browser: any
 - Version: latest PF5 as of 9/14/23

**Any other information?**
I also tried just importing `@patternfly/patternfly` and noticed that when using this import from within a tsx file `import "@patternfly/patternfly/patternfly.css";` it causes a similar build failure

```
🚨 Build failed.

@parcel/transformer-css: Unexpected token Ident("false")

  /path/to/project/node_modules/@patternfly/patternfly/patternfly.css:6871:14
    6870 | :where(.pf-v5-theme-dark) .pf-v5-c-banner {
  > 6871 |   color: var(false);
  >      |              ^
    6872 | }
    6873 | :where(.pf-v5-theme-dark) .pf-v5-c-banner.pf-m-blue, :where(.pf-v5-theme-dark) .pf-v5-c-banner.pf-m-red, :where(.pf-v5-theme-dark) .pf-v5-c-banner.pf-m-green, :where(.pf-v5-theme-dark) .pf-v5-c-banner.pf-m-gold {
```

Switching that import to `import "@patternfly/patternfly/patternfly-base.css";` allowed the build to succeed, but it doesn't style the raw pf component markup.

<img width="243" alt="Screenshot 2023-09-14 at 9 47 44 AM" src="https://github.com/patternfly/patternfly-react/assets/5942899/cfa2531a-a522-44d9-84e2-51637dd072e9">

Maybe I'm missing something config-wise for parcel (I'm new to using that) - if there's any special config/setup we need to do when using pf with parcel, it would be cool to have an example or doc somewhere. Maybe something to include in the seed repo as a wiki or something? Thanks in advance for any insights!

**Comment by mcoker** [comment]
Hey @seanforyou23! Thanks for filing this issue. FWIW the invalid CSS causing that build failure was fixed/merged this morning in https://github.com/patternfly/patternfly/pull/5933, so that should take care of the build error in our upcoming release.

**Comment by seanforyou23** [comment]
Hey @mcoker! Oh snap, that's great! Thanks for the quick update - I'll pull the latest and see if I can get it going - been dying to test drive the new PF5 stuffs :)

**Comment by seanforyou23** [comment]
Quick update, I pulled `@patternfly/patternfly@5.1.0-prerelease.28` and the issue is fixed, I get styles!

I also tried pulling in `@patternfly/react-core@5.1.1-prerelease.11` to see if it by chance would work in the react-styles package, but get the same error as before.

I guess I just need to be patient and wait for another release to be cut :) I'll sit tight and try again when I see a new prerelease version to come. Any idea when these typically go out? No rush, just curious. Thank you!

**Comment by mcoker** [comment]
Thank you for reporting the issue and being involved in the community!! You're continuing to make PF great! 🍻

We've got a PR in the react repo to bump core to the version with this fix, just needs to be merged. Then it will be available in a react pre-release version. https://github.com/patternfly/patternfly-react/pull/9601

We have moved to a 3 month/quarterly release cycle,  and are wrapping up the last sprint for the quarter as we speak, and are set to release once we've finished testing. I'd say we're hoping to publish a new version with this fix to the latest stream next week or the week after?
